x-pack/plugins/cases/public/containers/use_delete_cases.tsx

```diff
@@ -85,11 +85,9 @@ export const useDeleteCases = (): UseDeleteCase => {
       dispatch({ type: 'FETCH_INIT' });
 
       const caseIds = cases.map((theCase) => theCase.id);
-      // We don't allow user batch delete sub cases on UI at the moment.
-      if (cases[0].type != null || cases.length > 1) {
```

Comment:
@cnasikas shouldn't this be `cases.length > 0`?

Reply:
Tbh I don't remember why there was this check. I think you are right and the new logic should be `cases.length > 0`.